core/invoker/src/main/scala/whisk/core/container/AkkaHttpUtils.scala

```diff
@@ -24,24 +24,38 @@ import scala.concurrent.duration.DurationInt
 import scala.concurrent.duration.FiniteDuration
 import scala.util.Try
 import akka.actor.ActorSystem
-import akka.http.scaladsl._
 import akka.http.scaladsl.model._
 import akka.stream.ActorMaterializer
-import akka.stream.scaladsl._
 import java.util.concurrent.TimeoutException
 
+import play.api.http.{ContentTypeOf, Writeable}
```

Comment:
It is subject to investigation, yes. But I don't think it'll happen in the next weeks. I can just refactor back if I need it.